backend/controllers/cardsController.js

```diff
@@ -0,0 +1,25 @@
+const User = require('../UserModel');
+const cardsController = {};
+
+cardsController.getCards = async (req, res, next) => {
+  //retrieve cards collection data from DB
+  const cards = await User.findOne({ username });
+  //pass cards collection data into res.locals
+  res.locals.cards = cards;
+  return next();
+};
+
+
+cardsController.deleteCard = async (req, res, next) => {
+  //delete card specific to a user from DB
+  await User.findOneAndDelete({ username, cardID });
```

Comment:
Will this delete the card or the user?

It looks like it will delete the use rather than the card. I could be wrong though.